### PR TITLE
feat: pairings screen

### PR DESCRIPTION
**Pull request by HelenGezahegn**
**Changes**
- deleted `flutter_tindercard` package
- added ano on mountain image
- added pairings page that shows current card with dummy data

**UI**
_Used the android emulator because my iOS one wasn't working_
> Pixel 
> 

> Pixel 2 XL
>

### CODE REVIEW

File: lib/pairings/pairings.dart [hunk 1]
```diff
@@ -1,21 +1,155 @@
...
+  Container buildCards(context) {
+    return Container(
+      width: screenWidth * 0.80,
+      height: screenHeight * 0.42,
+      decoration: buildCardContainer(context), 
+      child: Stack(
+        overflow: Overflow.visible, 
+        alignment: Alignment.center, 
+        children: [
+          buildCurrentCard(context),
```
Comment (HelenGezahegn) [inline lib/pairings/pairings.dart:145]: I was thinking once I add the database calls to load potential matches in this file, I would move a lot of these functions into a `Cards` widget and use the animation slider package to render/animate all the cards.

File: lib/pairings/pairings.dart [hunk 2]
```diff
@@ -1,21 +1,155 @@
 import 'package:flutter/material.dart';
 
+import '../common/common.dart';
 import '../common/global_header.dart';
-import './pairings_card.dart';
+import '../constants/constants.dart';
+
+double screenWidth;
```
Comment (HelenGezahegn) [inline lib/pairings/pairings.dart:7]: Decided to make these global!

File: lib/pairings/pairings.dart [hunk 3]
```diff
@@ -1,21 +1,155 @@
...
+    return Image.asset(
+      'images/ano_mountains.png',
+      height: screenHeight * 0.25
+    );
+  }
+
+  PrimaryButton buildReadMoreButton() {
+    return PrimaryButton(
+      isLight: true,
+      text: readMoreAction,
```
Comment (HelenGezahegn) [inline lib/pairings/pairings.dart:44]: Should I rename this?

Comment (xiangxinyue) [reply lib/pairings/pairings.dart:44]: Yes, I think so.

Comment (HelenGezahegn) [reply lib/pairings/pairings.dart:44]: to readMoreActionText?

**Pull request merged**